### DISCUSSION TITLE
Path of local notes

### DISCUSSION THREAD

**Discussion by azizsayfiddin** [maelchiotti/LocalMaterialNotes#179] [Q&A] [answered]
Where Are local notes stored?

**Comment by maelchiotti** [comment, marked as answer]
Notes are stored using the [isar](https://github.com/isar/isar) database. I configured it to store its database file in the application's internal storage, which is provided by Android for all apps (`data/user/0/com.maelchiotti.localmaterialnotes`).

However, access to this directory is heavily restricted by Android, I'm pretty sure you can't access it without rooting your phone or using debugging via USB.

Even if you do that, the database file is a binary file so it can't be used outside of the app.
